content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -418,3 +418,63 @@ The cash drawer should be connected to the printer and the :guilabel:`Cash drawe
 be ticked in the :abbr:`PoS (Point of Sale)` configuration. To do so, navigate to
 :menuselection:`POS app --> 3-Dot Menu on the POS --> IoT Box section --> Edit --> Receipt Printer
 --> Cashdrawer checkbox`.
+
+Scale
+=====
+
```

Comment:
Is there an introductory "bumper" sentence we could put between these two headers? It's not entirely necessary, just nice to have